[UG] Branch enhance ug (#207)
AY2223S2-CS2103T-W14-1/tp: docs/UserGuide.md

---

> :mushroom:Mycelium is a desktop application aimed at helping **freelance web developers

Reviewer: I wonder if the 🍄 is excessive

### Creating a client contact: `c`

> To create a new client contact, use the **`c`** command followed by the client's name and email. 

Reviewer: This may lead to people misunderstanding as `c ProjectName`. May want to say as compulsory arguments

Reviewer: Do we need to bold the `c`?

Reviewer 2: Hmm, i also feel like what prof recommended us to do is to use more 'you' language, for example here we can rephrase to 'You can create a new client contact by using the ...'

## Managing Projects

> This section of the user guide explains how to _create_, _delete_, and _update_ a project using commands that allows users to manage their projects efficiently.

Reviewer 3: ```suggestion
This section of the user guide explains how to _create_, _delete_, and _update_ a project using commands that allow you to manage your projects.
```

### Fuzzy searching projects

We will now demonstrate fuzzy searching through an example. Let us assume we
have the following projects in :mushroom:Mycelium:

<img src="images/FuzzyDemoInitial.png" />

<div markdown="span" class="alert alert-info">

> :information_source: Recall that :mushroom:Mycelium supports fuzzy search for projects by

Reviewer: Maybe u want to move this to before the image, since the numbered point is attached to the image

> their *names* only!

Reviewer 3: ```suggestion
their *names* only.
```

</div>

> 1. First, press (CTRL+F) to toggle to **search mode** (if you are not already in

Reviewer 3: hm feel like this way of numbering is kinda weird. i'll prolly touch up on it again later

search mode). You should see the command box turn teal; now we can begin
searching. Suppose we wanted to search for *Clash of Clans* listed at the
bottom in the screenshot above. With the power of fuzzy matching, just typing
*coc* is enough, as shown below.

<img src="images/FuzzyDemoSearching.png" />

> - If we queried a term which matches nothing at all, then no results will be

Reviewer: Is there a reason for choosing to use `-`?

Author: I used the `-` as a precursor for a notice instead of numbering them which i did for sentences which are more instructional

listed.

<img src="images/FuzzyDemoSearchingNoResults.png" />

> 2. Once we are done, pressing (CTRL+F) again switches us back to command mode.

Reviewer 3: ```suggestion
2. To exit search mode, press (CTRL+F) again. The command box should return to its original dark color.
```
